source/fundamentals/crud/read-operations/project.txt

```diff
@@ -0,0 +1,196 @@
+==============================
+Specify Which Fields to Return
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return in a
+document from read operations.
+
+Sample Data
+~~~~~~~~~~~
+
+To run the examples in this guide, load these documents into the
+``ratings`` collection of the ``tea`` database with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/projection.go 
+   :language: go
+   :dedent:
+   :start-after: begin insertDocs
+   :end-before: end insertDocs
+
+.. include:: /includes/fundamentals/tea-sample-data-ending.rst
+
+Projection
+----------
+
+A projection specifies which fields to return in a document. To create a
+projection, pass an interface specifying the field and whether to
+indclude that field to the ``SetProjection()`` function of a read
+operation's options. To include the field, specify a ``1`` and to
+exclude a field, specify a ``0``. 
```

Comment:
This paragraph is kinda difficult to grok. The description of the structure of the projection document and usage in the Go driver API are much better illustrated in the examples below and can be excluded here.

Example:
> A [projection](https://docs.mongodb.com/manual/reference/glossary/#std-term-projection) is a document that specifies which fields to return in query results. A projection document includes field names followed by a ``1`` (to include) or ``0`` (to exclude). See the examples below for more details on the projection document format and how to use projections in the Go driver.